Subject: [PATCH] Do not include <asm/perfmon_kern.h> if !CONFIG_PERFMON

When compiling for an architecture that doesn't support perfmon yet (e.g.
ARCH=um), I get:

    In file included from kernel/sched.c:73:
    include/linux/perfmon_kern.h:508:30: error: asm/perfmon_kern.h: No such file or directory

Hence do not include <asm/perfmon_kern.h> if CONFIG_PERFMON is not set.

Signed-off-by: Geert Uytterhoeven <Geert.Uytterhoeven@sonycom.com>
Signed-off-by: Stephane Eranian <eranian@googlemail.com>
---
 include/linux/perfmon_kern.h |    5 -----
 1 file changed, 5 deletions(-)

--- a/include/linux/perfmon_kern.h
+++ b/include/linux/perfmon_kern.h
@@ -501,11 +501,6 @@ void pfm_cpu_disable(void);
 #define pfm_session_allcpus_release()	do { } while (0)
 #define pfm_session_allcpus_acquire()	(0)
 
-/*
- * include arch-specific kernel level definitions
- */
-#include <asm/perfmon_kern.h>
-
 #endif /* CONFIG_PERFMON */
 
 #endif /* __KERNEL__*/
